alembic/versions/2024_03_18_24c1e9e1476d_remove_unused_sample_columns.py

```diff
@@ -0,0 +1,29 @@
+"""Remove unused sample columns
+
+Revision ID: 24c1e9e1476d
+Revises: b2fcb2ada306
+Create Date: 2024-03-18 14:59:13.832047
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "24c1e9e1476d"
+down_revision = "b2fcb2ada306"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_column("sample", "sequence_start")
+    op.drop_column("sample", "is_external")
```

Comment:
We do have samples with is_external set although the field is deprecated. Maybe okay to drop but good to double check?:

<img width="236" alt="image" src="https://github.com/Clinical-Genomics/cg/assets/40887124/bee5890c-b926-41c9-aa26-8db13a9b6ff7">

Reply:
Good spot 👍

Reply:
All of them are from 2016/2017 so I don't think it is a problem to remove the column.